Example/package.json

```diff
@@ -24,7 +24,7 @@
     "react-native-gesture-handler": "^2.13.2",
     "react-native-pager-view": "^5.4.25",
     "react-native-reanimated": "link:../",
-    "react-native-safe-area-context": "^4.7.2",
+    "react-native-safe-area-context": "4.7.4",
```

Comment:
1. Is it related to KeaboardAvoidingView>
2. In some of our examples we use safeAreaView from "react-native-safe-area-context" in others from "react-native", so we will have to unify it.